src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleanerTest.java

```diff
@@ -0,0 +1,175 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.processors;
+
+import com.azure.core.test.http.MockHttpResponse;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.models.BlobStorageException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.data.EnvelopeRepositoryImpl;
+import uk.gov.hmcts.reform.blobrouter.data.model.Envelope;
+import uk.gov.hmcts.reform.blobrouter.data.model.Status;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Status.DISPATCHED;
+
+@ExtendWith(MockitoExtension.class)
+class ContainerCleanerTest {
+
+    private static final String CONTAINER_NAME = "bulkscan";
+
+    private ContainerCleaner containerCleaner;
+
+    @Mock
+    private EnvelopeRepositoryImpl envelopeRepository;
+
+    private BlobServiceClient storageClient = mock(BlobServiceClient.class);
+
+    @Mock
+    private BlobContainerClient containerClient;
+
+    @Mock
+    private BlobClient blobClient1;
+
+    @Mock
+    private BlobClient blobClient2;
+
+    @Mock
+    private BlobClient blobClient3;
+
+    @Mock
+    private BlobClient blobClient4;
+
+    private static final UUID UUID_1 = UUID.randomUUID();
+    private static final UUID UUID_2 = UUID.randomUUID();
+
+    private static final String FILE_NAME_1 = "file1.zip";
+    private static final String FILE_NAME_2 = "file2.zip";
```

Comment:
Those constants can be removed.
We can refer to fields from `ENVELOPE_1` and `ENVELOPE_2` instead.
It will also clarify what those are in my opinion.

Reply:
File names constants are used